app/lib/nomis_client/image.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module NomisClient
+  # Sadly can't get VCR to work (yet) so can't auto-test this
+  # :nocov:
+  class Image < NomisClient::Base
+    class << self
+      def get booking_id
+        image_route = "/bookings/#{booking_id}/image/data"
+        begin
+          NomisClient::Base.get(image_route).body
+        rescue OAuth2::Error => e
+          # We are currently calling Faraday via the OAuth2::Client which
+          # results in it wrapping errors inside an OAuth2::Error object
+          if e.response.status == 404
+            nil
+          else
+            raise
```

Comment:
this exception is generic,
we can add some useful info here:
raise("Image not found for the booking_id: xxxx")

Reply:
No we can't. At this point we have no idea what went wrong, hence the re-raise